lib/lure/command/updateDependencies.go

```diff
@@ -0,0 +1,217 @@
+package command
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"strings"
+
+	"github.com/coveooss/lure/lib/lure"
+	"github.com/coveooss/lure/lib/lure/log"
+	"github.com/coveooss/lure/lib/lure/project"
+	p "github.com/coveooss/lure/lib/lure/provider"
+	"github.com/coveooss/lure/lib/lure/versionManager"
+	"github.com/coveooss/lure/lib/lure/versionManager/mvn"
+	"github.com/coveooss/lure/lib/lure/versionManager/npm"
+
+	"github.com/vsekhar/govtil/guid"
+)
+
+// This part interesting
+// https://github.com/golang/go/blob/1441f76938bf61a2c8c2ed1a65082ddde0319633/src/cmd/go/vcs.go
+
+func appendIfMissing(modules []versionManager.ModuleVersion, modulesToAdd []versionManager.ModuleVersion) []versionManager.ModuleVersion {
+	for _, moduleToAdd := range modulesToAdd {
+		exist := false
```

Comment:
You could split this whole "for loop" into its own `func moduleExist`. Then you could basically have this in "appendIfMissing":

```
if !moduleExist(modules, moduleToAdd) {
    modules = append(modules, moduleToAdd)
}
```